### ISSUE TITLE
JavaScript error on launch - Mac

### ISSUE THREAD

**Issue by Taffer** [sedwards2009/extraterm#131]
The second and subsequent times I launch Extraterm on Mac OS (10.14 "Mojave") I get a JavaScript error. Clicking "OK" (the only button in the error dialog) closes the dialog, leaves the app running, but I don't get a window, and the only option in the menu is Quit.

```
Uncaught Exception:
TypeError: Path must be a string. Received undefined
    at assertPath (path.js:28:11)
    at basename (path.js:1395:5)
    at Command.parse (/private/var/folders/jd/t2rlxz8525z515_1cx1jg9t85q8b2g/T/AppTranslocation/A4968BAF-CD7B-447F-9E56-7F1E98BA8CAE/d/Extraterm.app/Contents/Resources/app/node_modules/commander/index.js:462:30)
    at main (/private/var/folders/jd/t2rlxz8525z515_1cx1jg9t85q8b2g/T/AppTranslocation/A4968BAF-CD7B-447F-9E56-7F1E98BA8CAE/d/Extraterm.app/Contents/Resources/app/extraterm/src/main_process/Main.js:95:10)
    at Object.<anonymous> (/private/var/folders/jd/t2rlxz8525z515_1cx1jg9t85q8b2g/T/AppTranslocation/A4968BAF-CD7B-447F-9E56-7F1E98BA8CAE/d/Extraterm.app/Contents/Resources/app/extraterm/src/main_process/Main.js:1067:1)
    at Object.<anonymous> (/private/var/folders/jd/t2rlxz8525z515_1cx1jg9t85q8b2g/T/AppTranslocation/A4968BAF-CD7B-447F-9E56-7F1E98BA8CAE/d/Extraterm.app/Contents/Resources/app/extraterm/src/main_process/Main.js:1069:3)
    at Module._compile (module.js:642:30)
    at Object.Module._extensions..js (module.js:653:10)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
```

Deleting all the Extraterm files and unpacking them again from the release `.zip` file lets me run it once successfully.

**Comment by sedwards2009** [comment]
Thanks for reporting this. It has been fixed in 0.37.2 which is just released.

**Comment by Taffer** [comment]
Excellent, thanks!
